contrib/PKGBUILD

```diff
@@ -0,0 +1,35 @@
+# Maintainer: Bruno Pagani (a.k.a. ArchangeGabriel) <archange@archlinux.org>
+# Contributor: Mirco Tischler <mt-ml at gmx dot de>
+
+pkgname=fwupd
+pkgver=LATEST
```

Comment:
Hum… Not sure what you want here (but in any case I must change at least one thing here).

We can indeed detect the version from the git tag (or parse from the meson file, but they’re supposed to be the same, right?), but then you’re building an artifact with a fixed version (the latest tag) while the content is moving (latest commit). Sure, when building at the tag, it would match. Is that what you want?

Note that I don’t see what purpose would serve the built artifact though (I would not be able to push it in Arch, and even if I could I would still need to build i686 package).

The other options are:

- to build a standard -git package in the Arch meaning of that: the version would then be something like 0.9.5+r24:a74efb8 (last tag, number of commits since, latest commit).
- to build a dummy debug package, with version LATEST, DEBUG or whatever.

For both, I would change the name to fwupd-git and fwupd-debug respectively.